src/Stratis.Bitcoin.IntegrationTests.Common/Extensions/CoreNodeExtensions.cs

```diff
@@ -8,6 +9,14 @@ namespace Stratis.Bitcoin.IntegrationTests
 {
     public static class CoreNodeExtensions
     {
+        public static void AppendToConfig(this CoreNode node, string configKeyValueItem)
```

Comment:
Did you intent you going to use this anywhere?

Reply:
Good point, sorry about this. It was potentially going to be used but the test got changed.

I think it makes sense to put it here anyhow, but if it's a problem let me know

Reply:
All good, we can definitely use this!